app/frontend/components/common/viewPort.tsx

```diff
@@ -31,8 +31,8 @@ const isSmallerThanDesktop = (screenType: ScreenType): boolean => {
   return [ScreenType.MOBILE, ScreenType.TABLET].includes(screenType)
 }
 
-const isBiggerThanMobile = (screenType: ScreenType): boolean => {
-  return [ScreenType.TABLET, ScreenType.DESKTOP].includes(screenType)
+const isBiggerThanMobile = (screenType: ScreenType | undefined): boolean => {
+  return !!(screenType && [ScreenType.TABLET, ScreenType.DESKTOP].includes(screenType))
```

Comment:
I'd rather keep the interface of this function oblivious to the fact screenType might be undefined and fix the upstream logic to handle that

Reply:
This could also be handled by setting a default value for the state managed by the `useViewport` hook. Which would be similar to setting a fallback when the caller checks if the value is undefined or not.